esprima.js

```diff
@@ -198,7 +198,10 @@
         StrictLHSAssignment: 'Assignment to eval or arguments is not allowed in strict mode',
         StrictLHSPostfix: 'Postfix increment/decrement may not have eval or arguments operand in strict mode',
         StrictLHSPrefix: 'Prefix increment/decrement may not have eval or arguments operand in strict mode',
-        StrictReservedWord: 'Use of future reserved word in strict mode'
+        StrictReservedWord: 'Use of future reserved word in strict mode',
+        ParameterAfterRestParameter: 'Rest parameter must be final parameter of an argument list',
+        DefaultRestParameter: 'Rest parameter can not have a default value',
+        ObjectPatternAsRestParameter: 'Invalid rest parameter'
```

Comment:
https://chromium.googlesource.com/v8/v8.git/+/master/src/messages.js is a good source, beside the usual hassle of checking out and building V8.